src/Bynder/Api/Impl/AbstractRequestHandler.php

```diff
@@ -5,15 +5,20 @@ abstract class AbstractRequestHandler
 {
     protected $configuration;
 
-    public function sendRequestAsync($requestMethod, $uri, $options = [])
-    {
+    public function formRequest(&$uri, $requestMethod) {
```

Comment:
Not a big fan of methods with side effects. Would rather let it return the composed uri.

And I don't understand what we would win over the old situation where this was called `sendRequestAsync`?

Reply:
gotcha